corehq/apps/reports/templates/reports/partials/scheduled_reports_table.html

```diff
@@ -0,0 +1,135 @@
+{% load case_tags %}
+{% load hq_shared_tags %}
+{% load i18n %}
+{% load report_tags %}
+{% load compress %}
+{% if scheduled_reports %}
+<table class="table table-striped table-bordered">
+    {% if not is_owner %}
+    <h3>Other Scheduled Reports (managed by others)</h3>
+    {% endif %}
+    <thead>
+        <tr>
```

Comment:
Can you put an extra column here, only for the `others_scheduled_reports` table, called "Owner" that displays the name of the owner of that report? I noticed while testing on staging that you still couldn't easily tell who created a given report (and thus who you can ask to edit it).